release_notes_generator/model/record.py

```diff
@@ -164,7 +167,8 @@ def contains_release_notes(self) -> bool:
         if self.__is_release_note_detected:
             return self.__is_release_note_detected
 
-        if RELEASE_NOTE_LINE_MARK in self.get_rls_notes():
+        rls_notes = self.get_rls_notes()
```

Comment:
I would add a type hint, that this method returns a string.

Reply:
Fixed in commit - 5eee2e95bf800418adf6070e6d77d1882d4bf103.